stdlib/asyncio/events.pyi

```diff
@@ -599,8 +599,10 @@ class AbstractEventLoopPolicy:
     def new_event_loop(self) -> AbstractEventLoop: ...
     # Child processes handling (Unix only).
     @abstractmethod
+    @deprecated("Deprecated as of Python 3.12; will be removed in Python 3.14")
```

Comment:
Wasn't sure about these child watchers as it's not clear to me what if anything they should be replaced with. Possibly we should emit the warning on 3.12+ only.

Reply:
I haven't looked at this in detail, but in situations like this where we're not sure, I feel like we should probably err on the side of caution and only emit warnings on py312+

Reply:
Changed this to warn only on 3.12+